package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@react-native-community/art",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
You shouldn't increment this version number. I will increase it during the release.